src/main/java/net/datafaker/base/Internet.java

```diff
@@ -151,6 +151,14 @@ public String password(int minimumLength, int maximumLength, boolean includeUppe
 
         return faker.password().password(config);
     }
+    
+    /**
+     * Returns a port number between 0 and 65535
+     * @return a port number
+     */
+    public String port() {
```

Comment:
why should be string?

Reply:
I hastily did, thought for a moment that port should be a String in order to follow the format, after your comment realized that it's not true.

Reply:
Changed to int